src/main/java/gregtech/common/tileentities/generators/GT_MetaTileEntity_MagicalEnergyAbsorber.java

```diff
@@ -38,11 +39,13 @@
     public static boolean sAllowMultipleEggs = true;
     public static GT_MetaTileEntity_MagicalEnergyAbsorber mActiveSiphon = null;
     public static int sEnergyPerEnderCrystal = 32;
-    public static int sEnergyFromVis = 12800;
+    public static int sEnergyFromVis = 512; //really? this high? no wonder no eu was generating... (it was 12800 at the time)
     public static int sDragonEggEnergyPerTick = 128;
     public static boolean isThaumcraftLoaded;
+    public static boolean tLameMode = false; // if you use this... lame.
     public int mEfficiency;
     public EntityEnderCrystal mTargetedCrystal;
+    //public static int[] VoltageOut = {32, 128,512,2048}; // Because what the fuck even is this code.
```

Comment:
What is this doing here?

Reply:
Ugh I'm really sorry I left those things in when I was testing and forgot to remove them. I will fix those as soon as possible.